scripts/validateRelease.ts

```diff
@@ -17,7 +17,7 @@ npm run validateRelease -- --publishTag preview --dropPath /example/path/`);
 }
 
 function validateRelease(publishTag: string, dropPath: string): void {
-    const files = readdirSync(dropPath);
+    const files = readdirSync(dropPath).filter((f) => f.endsWith('.tgz'));
```

Comment:
1es automatically downloads the sbom manifest in addition to the drop contents, which messed up this validation script